src/DotVVM.Framework/Compilation/DefaultViewCompilerCodeEmitter.cs

```diff
@@ -167,7 +169,7 @@ public string EmitCreateObject(string typeName, object[] constructorArguments =
             }
 
             var typeSyntax = ReflectionUtils.IsFullName(typeName)
-                ? ParseTypeName(ReflectionUtils.FindType(typeName))
+                ? ParseTypeName(compiledAssemblyCache.FindType(typeName))
```

Comment:
When I see this, the type parsing should not be done by emitter, can we move it to the caller?